File: Java/Arrays/ArrayBasics.java
public class ArrayBasics { 
    public static void main(String[] args){
        //creating an array O(1)
        int[] numbers = {5, 3, 9, 1, 4};

        //access O(1)
        System.out.println("Third Element: " + numbers[2]);

        //update O(1)
        numbers[2] = 81;
        System.out.println("Updated Third Element: " + numbers[2]);

        //traverse O(n)
        System.out.println("Array: ");
        for(int num : numbers) {
            System.out.println(num + " ");
        }

        //finding min and max O(n)
        int min = numbers[0];
        int max = numbers[0];
        for(int i = 0; i < numbers.length; i++){
            if(numbers[i]>max)max=numbers[i];
            if(numbers[i]<min)min=numbers[i];
        }
        System.out.println("Max : "+ max);
        System.out.println("Min : "+ min);
    }
}

File: Python/Arrays/ArrayBasics.py
#declare and initialize
numbers = [5, 3, 9, 1, 4]

#access
print("Third Element: ", numbers[2])

#update
numbers[2] = 81
print("Updated Third Element: ", numbers[2])

#traverse
print("Array : ")
for num in numbers: 
    print(num, " ")

#insert
pos = 2
print("Inserting value(20) at position(2)")
numbers.insert(2, 20)
print("Array : ")
for num in numbers: 
    print(num, " ")
    
#delete
print("Deleting 20 of index2")
del numbers[2]
print("Array : ")
for num in numbers: 
    print(num, " ")